test/markdown-help.test.js

```diff
@@ -0,0 +1,47 @@
+/* global expect */
+/* eslint-disable func-names, prefer-arrow-callback */
+/* eslint import/no-extraneous-dependencies: ["error", { "devDependencies": true  }] */
+import React from 'react';
+import { mount } from 'enzyme';
```

Comment:
Let's use `shallow` instead of `mount`. The [first answer](https://stackoverflow.com/questions/38710309/when-should-you-use-render-and-shallow-in-enzyme-react-tests) on this stackoverflow question has a good explanation.